src/main/java/core/basesyntax/dao/ManufacturerJdbcDaoImpl.java

```diff
@@ -0,0 +1,123 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.exception.DataProcessingException;
+import core.basesyntax.lib.Dao;
+import core.basesyntax.model.Manufacturer;
+import core.basesyntax.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerJdbcDaoImpl implements ManufacturerDao {
+    private static final int INDEX_ONE = 1;
+    private static final int INDEX_TWO = 2;
+    private static final int INDEX_THREE = 3;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufacturers (manufacturer_name, "
+                + "manufacturer_country) VALUE (?,?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement createdStatement = connection
+                    .prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
+            createdStatement.setString(INDEX_ONE, manufacturer.getName());
+            createdStatement.setString(INDEX_TWO, manufacturer.getCountry());
+            createdStatement.execute();
+            ResultSet resultSet = createdStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                manufacturer.setId(resultSet.getLong(INDEX_ONE));
```

Comment:
be attentive with: https://mate-academy.github.io/jv-program-fulltime/02_jdbc/checklist/05_JBDC_checklist.html